app/components/Table.tsx

```diff
@@ -107,6 +107,7 @@ interface Props {
   onPrev?: (val: number) => void;
   onNext?: (val: number) => void;
   loading?: boolean;
+  pageLimitBox?: string;
```

Comment:
this won't be needed if we have that conditional

Reply:
removed